Buttplug.Server/Bluetooth/Devices/Kiiroo.cs

```diff
@@ -45,8 +46,15 @@ internal class Kiiroo : ButtplugBluetoothDevice
                    aInterface,
                    aInfo)
         {
-            MsgFuncs.Add(typeof(KiirooCmd), HandleKiirooRawCmd);
-            MsgFuncs.Add(typeof(StopDeviceCmd), HandleStopDeviceCmd);
+            MsgFuncs.Add(typeof(KiirooCmd), new ButtplugDeviceWrapper(HandleKiirooRawCmd));
+            MsgFuncs.Add(typeof(StopDeviceCmd), new ButtplugDeviceWrapper(HandleStopDeviceCmd));
+
+            if (aInterface.Name == "PEARL")

```

Comment:
At this point I'd argue we should divide into KiirooOnyx and KiirooPearl classes, but I realize we already kind of multiplex devices in the lovense handler, so that can go in a followup for now.

Reply:
Again, there's so much overlap, it seems wasteful. But these may well diverge more as I actually get the Onyx to work.